daisy/README.md

```diff
@@ -1,28 +1,141 @@
-# Daisy
+Definitions:
+* GCP: Google Cloud Platform
+* GCS: Google Cloud Storage
+* GCE: Google Compute Engine
+* Workflow: a graph of executable, blocking steps and their dependency
+relationships
 
-Daisy is a GCE workflow tool. Development is ongoing.
+# What is Daisy?
+Motivated by Google-internal image build workflows, Daisy is a solution for
```

Comment:
Remove the motivated part...
Daisy is a solution for running complex, multi-step workflows for GCE.